README.md

```diff
@@ -92,3 +93,44 @@ More useful commands to execute from the `data-science-101` root directory
 - `vagrant destroy -f && vagrant up`
   - if things get SUPER messed up and you need to start all over
 
+
+## Alternative setup using docker 
+
+### Step 1
+
+Install:
+- [git](https://git-scm.com/)
+  - If you're on Windows or OS X and don't know what git is, [GitHub Desktop](https://desktop.github.com/) is probably easiest.
+  - All others: you probably know what to do.
+- [docker](https://docs.docker.com/engine/installation/)
+
+### Step 2
+
+If you installed git using `GitHub Desktop`, follow [these instructions](https://help.github.com/desktop/guides/contributing/cloning-a-repository-from-github-to-github-desktop/)
+
+If you are cloning from the command line:
+```shell
+1. $ git clone https://github.com/hershaw/data-science-101.git
+```
+
+### Step 3 
+
+1. Build a docker image (first time takes longer):
+```shell
+    $ docker build -t data-science -f Dockerfile .
+```
+2. Run a container:
+```shell
+    $ docker run -it --rm -p 8888:8888 --volume ~/workspace/data-science-101:/data-science-101 \
+    --workdir /data-science-101 data-sciencecience-101 -e -e PYTHONPATH=.:/data-science-101/course data-science
+```
+
+### Step 4
+
+Open the link provided in the interactive shell something like http://localhost:8888/?token=d1ab96467698d1addffac95d634d777fd059ccd4a7b2094e.
```

Comment:
I could but then it would be unsafe.
Looking at netstat:
"tcp4       0      0  *.ddi-tcp-1            *.*                    LISTEN"
Decoding:
* stands for all IPs
* ddi-tcp-1 stands for port 8888

If the laptop does not have a firewall blocking incoming traffic on port 8888 then it will allow remote users to execute locally (on the docker) python code.

Still trying to figure out docker networking maybe it could be done with a private network like vagrant is doing.